README.md

```diff
@@ -231,71 +231,67 @@ Command above will build a vagrant box based on [fredrikhgrelland/bionic64-ansib
 The section also provides information about steps for overriding the default system configuration.
 
 ## Configuration
+The vagrant-box is set up with a lot of default configuration (see subsection [Default Configuration](#default-configuration)), and users may want to customize the vagrant-box services configuration to fit their need.
 
-In most cases, users need to customize vagrant-box's services configuration.
-
-Scenarios for customization:
-```text
-1. consul(enterprise) with consul_acl = enabled and default acl policy = deny
-2. nomad(oss) with nomad_acl = false
-3. vault(enterprise), unsealed and integrated with nomad and consul, the way that it manages their secrets/tokens
-```
+Say you want to do some tweaks and use Consul Enterprise, an open-source Nomad and Vault Enterprise. Then you would need to override the following variables:
+- Consul Enterprise: 
+    1. `consul_enterprise=true`
+- Nomad open source:
+    1. `nomad_enterprise=false`
+- Vault Enterprise:
+    1. `vault_enterprise=true`
```

Comment:
This should be moved down to where you explain overriding configuration. User has no knowledge of what this is just yet. 

```suggestion
```

Reply:
Done. See new changes

Reply:
Looking good!